projects/canopy/src/styles/variables/components/_card.scss

```diff
@@ -6,4 +6,8 @@
   --card-principle-data-point-label-color: var(--color-battleship-grey);
   --card-principle-data-point-date-color: var(--color-charcoal);
   --card-principle-data-point-value-color: var(--color-charcoal);
+  --card-header-margin-bottom: var(--space-lg);
+  --card-navigation-link-padding-bottom: 1.25rem;
```

Comment:
Do we need a css variable instead of `1.25rem`?

Reply:
The value is used in four different places within the card so I think it's good to have it in a variable 👍